src/index.js

```diff
@@ -116,3 +137,26 @@ deleteBtn.forEach((element, index) => {
     localStorage.setItem('lists', JSON.stringify(information));
   });
 });
+
+// completed
+
+const checkBox = document.querySelectorAll('.square');
+
+checkBox.forEach((element, index) => {
+  element.addEventListener('click', () => {
+    if (element.checked) {
+      boxChecked(element, index);
+    } else {
+      unCheck(element, index);
+    }
+  });
+});
+
+clearBtn.addEventListener('click', () => {
+  for (let i = 0; i < information.length; i++) clear(i);
+});
```

Comment:
- [ ] Please make sure that the "Clear all completed" button is working properly. Currently, when I tried to clear a couple of todos using the button, I got the following result.

![image](https://user-images.githubusercontent.com/62290545/208141032-d782376b-426a-4964-87fb-5dd6cc7479a3.png)